Gemfile

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+ruby RUBY_VERSION
+
+gemspec
+
+gem 'decidim', "~> 0.16.0"
```

Comment:
Update sitges app to `0.16-stable` so that we take advantage of all the bugfixes.

Reply:
Done